### PR TITLE
 Add image tag used by container for `crictl ps` and add attempt for `crictl ps/sandboxes`

### LINKED ISSUE #184: crictl ps should include image tag used by a container.
currently it is not.

### PR DESCRIPTION
**Pull request by miaoyq**
Fixes #184 #185 

/assign @Random-Liu 

Signed-off-by: Yanqiang Miao <miao.yanqiang@zte.com.cn>

### CODE REVIEW

File: cmd/crictl/container.go [hunk 1]
```diff
@@ -592,3 +594,12 @@ func ListContainers(client pb.RuntimeServiceClient, opts listOptions) error {
 	w.Flush()
 	return nil
 }
+
+func getImageRef(c *pb.Container) string {
```
Comment (Random-Liu) [inline cmd/crictl/container.go]: Actually in current CRI implementation `c.Image.Image` should mostly be image tag, I don't think you want to truncate it.
See:
* https://github.com/kubernetes-incubator/cri-containerd/blob/master/pkg/server/container_status.go#L49
* https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/dockershim/docker_container.go#L375


I think we could show the `c.Image.Image` directly.

**Comment by Random-Liu** [comment]
LGTM overall, please paste the output, rebase and address comments.

File: cmd/crictl/container.go (hunk 1 above)
Comment (miaoyq) [reply cmd/crictl/container.go]: OK, will do.

Comment (miaoyq) [reply cmd/crictl/container.go]: See: https://github.com/kubernetes/kubernetes/blob/08781adf13491e08ac25e7545a9099d8be73a9dd/pkg/kubelet/apis/cri/v1alpha1/runtime/api.proto#L443-L448

Still show `c.Image.Image` directly if `c.Image.Image` is imageID?

**Comment by miaoyq** [comment]
@Random-Liu  Updated. 
The output:
```bash
# crictl ps
CONTAINER ID        IMAGE                                                                                                               CREATED             STATE               NAME                ATTEMPT
d67cc2866acff       gcr.io/google_containers/kube-proxy-amd64@sha256:58fd496116bf3d2790279f9c44040bdfa71791aba7a7a1b5ca35025b5a032b9f   2 weeks ago         CONTAINER_RUNNING   kube-proxy          0
d7991a3ccae38       docker.io/library/busybox@sha256:3e8fa85ddfef1af9ca85a5cfb714148956984e02f00bec3f7f49d3925a91e0e7                   About an hour ago   CONTAINER_EXITED    busybox             147
757fc432b9e10       docker.io/library/busybox@sha256:3e8fa85ddfef1af9ca85a5cfb714148956984e02f00bec3f7f49d3925a91e0e7                   About an hour ago   CONTAINER_EXITED    busybox             280
c87e53334fc9b       docker.io/library/busybox@sha256:3e8fa85ddfef1af9ca85a5cfb714148956984e02f00bec3f7f49d3925a91e0e7                   23 minutes ago      CONTAINER_RUNNING   busybox             148
e3e809d23abaf       docker.io/library/busybox@sha256:3e8fa85ddfef1af9ca85a5cfb714148956984e02f00bec3f7f49d3925a91e0e7                   20 minutes ago      CONTAINER_RUNNING   busybox             281
```
```bash
# crictl sandboxes
SANDBOX ID          CREATED             STATE               NAME                NAMESPACE           ATTEMPT
41555a4bd665d       11 days ago         SANDBOX_READY       busybox             default             0
cce6f4f601d44       2 weeks ago         SANDBOX_READY       kube-proxy-gwb2k    kube-system         0
432374d9dfa61       6 days ago          SANDBOX_READY       busybox             myq                 0
```

**Pull request merged**